stor/cli.py

```diff
@@ -166,13 +167,19 @@ def _get_pwd(service=None):
     Returns the present working directory for the given service,
     or all services if none specified.
     """
+    def to_text(data):
+        if six.PY2:
+            return data.decode(locale.getpreferredencoding(False))
+        else:
+            return data
+
     parser = _get_env()
     if service:
         try:
-            return utils.with_trailing_slash(parser.get('env', service))
+            return to_text(utils.with_trailing_slash(parser.get('env', service)))
         except configparser.NoOptionError as e:
             six.raise_from(ValueError('%s is an invalid service' % service), e)
-    return [utils.with_trailing_slash(value) for name, value in parser.items('env')]
+    return [to_text(utils.with_trailing_slash(value)) for name, value in parser.items('env')]
```

Comment:
Don't fully understand this change yet.  Is there a bug report or test regarding this?

Reply:
ConfigParser generates bytes, but we want text

Reply:
it hits the assertionerror later on if you don't do this